src/mds/heartbeat/topo_updater.cpp

```diff
@@ -53,12 +55,13 @@ void TopoUpdater::UpdateTopo(const CopySetInfo &reportCopySetInfo) {
     // mds epoch fall behind, update needed:
     bool needUpdate = false;
     if (recordCopySetInfo.GetEpoch() < reportCopySetInfo.GetEpoch()) {
-        LOG(INFO) << "topoUpdater find report copyset("
-                  << reportCopySetInfo.GetLogicalPoolId()
-                  << "," << reportCopySetInfo.GetId()
-                  << ") epoch:" << reportCopySetInfo.GetEpoch()
-                  << " > recordEpoch:"
-                  << recordCopySetInfo.GetEpoch() << " need to update";
+        LOG(INFO) << "chunkserver " << reportCopySetInfo.GetLeader()
```

Comment:
above annotations item 3 should modify accordingly

Reply:
done

Reply:
done